examples/leader-example/src/main/scala/com/coralogix/zio/k8s/examples/leader/LeaderExample.scala

```diff
@@ -58,8 +59,10 @@ object LeaderExample extends App {
         Pods.live,
 //        ConfigMaps.live,
 //        LeaderElection.configMapLock("leader-example-lock"),
-        LeaderLockResources.live,
-        LeaderElection.customLeaderLock("leader-example-lock", deleteLockOnRelease = false)
+//        LeaderLockResources.live,
```

Comment:
Remove the commented out code?

Reply:
I left it on purpose to have a (commented out) example of all three methods

Reply:
ah ok sure